src/static/js/app/components/Survey.js

```diff
@@ -0,0 +1,92 @@
+import 'fetch';
```

Comment:
Nice, just had a great idea. I can pull all the calls to fetch out into a separate module. That way I can just mock that module and it doesn't have to worry about fetch... I really hope.